cluster/kubernetes/policies.go

```diff
@@ -89,21 +89,40 @@ func updateAnnotations(def []byte, f func(map[string]string) map[string]string)
 	return []byte(newDef), err
 }
 
-type annotationsParser struct {
-	Metadata struct {
-		Annotations map[string]string `yaml:"annotations"`
-	} `yaml:"metadata"`
+type Manifest struct {
+	Metadata Metadata `yaml:"metadata"`
+	Spec     struct {
+		Template struct {
+			Spec struct {
+				Containers []Container `yaml:"containers"`
+			} `yaml:"spec"`
+		} `yaml:"template"`
+	} `yaml:"spec"`
 }
 
-func parseAnnotations(def []byte) (map[string]string, error) {
-	var p annotationsParser
-	if err := yaml.Unmarshal(def, &p); err != nil {
-		return nil, errors.Wrap(err, "decoding annotations")
+func (m Metadata) AnnotationsOrNil() map[string]string {
```

Comment:
There's already some code for parsing manifests in `cluster/kubernetes/resources`, could it be used here _mutatis mutandis_?